src/mutex.hpp

```diff
@@ -182,15 +181,18 @@ class SCOPED_CAPABILITY UniqueLockGuardRO {
 public:
   UniqueLockGuardRO(t_mutex &mu) noexcept ACQUIRE_SHARED(m_mut) : m_mut(mu), m_locked(true) {
   	try { m_mut.lock_shared(); } catch(...) { _mutexshield_abort("LG-RO constr"); } }
-  ~UniqueLockGuardRO() noexcept RELEASE_SHARED() {
+//  ~UniqueLockGuardRO() noexcept RELEASE_SHARED() { // https://stackoverflow.com/questions/33608378/clang-thread-safety-annotation-and-shared-capabilities
+  ~UniqueLockGuardRO() noexcept RELEASE() {
```

Comment:
Need to make option for this workaround of broken clang  and comment with bugreport